Parsers/ASimUserManagement/Tests/Security_Event_ASimUserManagement_SchemaTest.csv

```diff
@@ -0,0 +1,72 @@
+﻿Result
+"(0) Error: Missing field [ActorUserIdType] is mandatory when field [ActorUserId] exists"
```

Comment:
can you please check this error and map ActorUserIDType?

Reply:
done

Reply:
@Alekhya0824 this seems still missing
![image](https://github.com/user-attachments/assets/7f05c13f-0bf1-4ae7-b5ef-c645056da1ac)

Also at:
![image](https://github.com/user-attachments/assets/1f4441c3-7ec9-4504-b779-45e3378a53f0)

Reply:
<img width="538" alt="image" src="https://github.com/user-attachments/assets/b4ccb26c-4a5e-4e15-84a7-1db8097aaa3e">

while i'm running in workspace i'm not getting this error